CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator+Camera.swift

```diff
@@ -73,14 +67,25 @@ extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
         }
     }
     
-    public func cameraDidTapMultipageReviewButton(_ viewController: CameraViewController) {
+    public func cameraDidTapMultipageReviewButton(_ viewController: CameraScreen) {
         showMultipageReview()
     }
     
-    func createCameraViewController() -> CameraViewController {
-        let cameraViewController = CameraViewController(giniConfiguration: giniConfiguration)
+    func createCameraViewController() -> CameraScreen {
+        let cameraButtonsViewModel = CameraButtonsViewModel(
+            trackingDelegate: trackingDelegate
+        )
+        cameraButtonsViewModel.helpAction = { [weak self] in
+            self?.showHelpMenuScreen()
+        }
+        cameraButtonsViewModel.cancelAction = { [weak self] in
+            self?.back()
+        }
+        let cameraViewController = Camera2ViewController(
+            giniConfiguration: giniConfiguration,
+            viewModel: cameraButtonsViewModel
+        )
         cameraViewController.delegate = self
-        cameraViewController.trackingDelegate = trackingDelegate
         cameraViewController.title = .localized(resource: NavigationBarStrings.cameraTitle)
```

Comment:
[Clarification] Why did we remove the line 
cameraViewController.trackingDelegate = trackingDelegate
?
We still need the tracking delegate 😊

Reply:
Oh I see it's in the Model 🙈